kubernetes/etc/kubernetes/k8s.node.rc

```diff
@@ -4,6 +4,6 @@ stuff "until nc -z __MASTER_IP__ 8080; do sleep 10; done; /home/ubuntu/kubernete
 screen -t kube-proxy
 select 1
 stuff "until nc -z __MASTER_IP__ 8080; do sleep 10; done; /home/ubuntu/kubernetes/cluster/ubuntu/binaries/minion/kube-proxy --master=http://__MASTER_IP__:8080^M"
-screen -t network-policy
+screen -t network-policy-agent
 select 2
-stuff "until curl -s http://__MASTER_IP__:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/ | grep -q policy; do sleep 10; done; python /home/ubuntu/romana/kubernetes/k8s-listener.py^M"
+stuff "until curl -s http://__MASTER_IP__:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/ | grep -q policy; do sleep 10; done; python /home/ubuntu/romana/kubernetes/k8s-listener.py --agent^M"
```

Comment:
I understand. Just wondering whether this should be part of the installation scripts, or whether this is something that the listener should be watching out for? Don't need to change this now, this will do, but in the long run I think it would be good to move this into the listener itself.